src/auro-combobox.js

```diff
@@ -282,7 +318,9 @@ class AuroCombobox extends LitElement {
             slot="trigger"
             borderless
             value="${this.displayValue === null ? `` : this.displayValue}"
-            ?required="${this.required}">
+            ?required="${this.required}"
+            .type="${this.type}"
+            ?icon="${this.triggerIcon}">
```

Comment:
There are now three different icons that can/will be in combobox at a time (e.g. this one, the clear X button for input and the chevron). I feel it's necessary to start make it clear which is which.

And `?icon="....` is necessary just for credit card because credit card is the only format where the icon is optional. This is the issue I brought up last week saying I think we need to make this a consistent toggle for all format types.